app/controllers/ems_common.rb

```diff
@@ -561,6 +563,10 @@ def button
     end
   end
 
+  def provider_documentation_url
+    "http://manageiq.org/documentation/getting-started/#adding-a-provider"
```

Comment:
The same-named method in an EMS model would *not* override this method, because this one is being the *controller*, not the *model*.